site/docs/v1/tech/core-concepts/tenants.adoc

```diff
@@ -42,6 +43,23 @@ This page describes the Admin UI for creating and configuring a Tenant.
 * <<Security>>
 * <<Advanced>>
 
+=== List of Tenants
+
+To display a list of tenants, navigate to [breadcrumb]#Tenants#.
+
+image::list-of-tenants.png[List of Tenants,width=1200,role=shadowed]
```

Comment:
No need for role of shadowed.

Please put this under the `site/assets/img/docs/core-concepts/` directory.

Reply:
And make sure you run it through tinypng after cropping (I crop with the Preview App on the mac and that increases the size of the PNG).